joomla/joomla-cms
# [4.3] Guided Tours improvement search label

### Summary of Changes

More meaningful label of search field for tours and step link e.g. for articles:
https://github.com/joomla/joomla-cms/blob/04b9ee83990a78ffc2df3ef1161ecc89bb30254b/administrator/language/en-GB/com_content.ini#L110

### Testing Instructions

Check label of search input for tours **and** steps
```html
<div class="input-group">
    <input type="text" name="filter[search]" id="filter_search" value="" class="form-control" aria-describedby="filter_search-desc" placeholder="Search" inputmode="search">
    <div role="tooltip" id="filter_search-desc" class="filter-search-bar__description">
        Search in title. Prefix with ID: to search for a tour ID or DESCRIPTION: to search in description.
    </div>
    <span class="filter-search-bar__label visually-hidden">
        <label id="filter_search-lbl" for="filter_search">
            Search Tours
        </label>
    </span>
    <button type="submit" class="filter-search-bar__button btn btn-primary" aria-label="Search">
        <span class="filter-search-bar__button-icon icon-search" aria-hidden="true"></span>
    </button>
</div>
```

### Link to documentations
Please select:
